### PR TITLE
create ez_40.mdx

### LINKED ISSUE #93: Create a page for `ez_40`
### description
Make `/docs/upgrades/locks/ez_40.mdx` using the [lock template](https://github.com/comcode-org/hackmud_wiki/blob/main/templates/lock.mdx) as a starter. Write a page about the lock that complies with the [content guide](https://github.com/comcode-org/hackmud_wiki?tab=readme-ov-file#content-guide).

### PR DESCRIPTION
**Pull request by kbinreallife**
fixes #93

### CODE REVIEW

File: docs/upgrades/locks/ez_40.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+## Behavior
+
+When encountered, the ez_40 will ask for a string on its first parameter, and a digit (number) on its second parameter.
+
+<details>
+<summary>Spoilers for ez_40's parameters</summary>
+```
+Incorrect parameters:
+
+> > user.fakeloc123 \{ ez_40: 1 }
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:22]: Remove space between `>>`

Remove the backslash in front of the bracket.

File: docs/upgrades/locks/ez_40.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+> > LOCK_ERROR
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:23]: This output shouldn't have `> >` on it. --remove

File: docs/upgrades/locks/ez_40.mdx (hunk 1 above)
Comment (seanmakesgames) [reply docs/upgrades/locks/ez_40.mdx:22]:
```suggestion
>> user.fakeloc123 { ez_40: 1 }
```

File: docs/upgrades/locks/ez_40.mdx (hunk 2 above)
Comment (seanmakesgames) [reply docs/upgrades/locks/ez_40.mdx:23]:
```suggestion
LOCK_ERROR
```

File: docs/upgrades/locks/ez_40.mdx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+> > 1 is not a string.
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:24]:
```suggestion
1 is not a string.
```

File: docs/upgrades/locks/ez_40.mdx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+> > user.fakeloc123 \{ ez_40: "release", ez_prime: "1" }
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:26]:
```suggestion
>> user.fakeloc123 { ez_40: "release", ez_prime: "1" }
```

File: docs/upgrades/locks/ez_40.mdx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+> > LOCK_ERROR
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:27]:
```suggestion
LOCK_ERROR
```

File: docs/upgrades/locks/ez_40.mdx [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+> > "1" is not a number.
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:28]:
```suggestion
"1" is not a number.
```

File: docs/upgrades/locks/ez_40.mdx [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+Correct parameters:
+
+> > user.fakeloc123 \{ ez_40: "release" }
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:32]:
```suggestion
>> user.fakeloc123 { ez_40: "release" }
```

File: docs/upgrades/locks/ez_40.mdx [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+> > LOCK_ERROR
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:33]:
```suggestion
LOCK_ERROR
```

File: docs/upgrades/locks/ez_40.mdx [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+> > Required unlock parameter ez_prime is missing.
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:34]:
```suggestion
Required unlock parameter ez_prime is missing.
```

File: docs/upgrades/locks/ez_40.mdx [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+
+> > user.fakeloc123 \{ ez_40: "release", ez_prime: 2 }
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:36]:
```suggestion
>> user.fakeloc123 { ez_40: "release", ez_prime: 2 }
```

File: docs/upgrades/locks/ez_40.mdx [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+> > LOCK_ERROR
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:37]:
```suggestion
LOCK_ERROR
```

File: docs/upgrades/locks/ez_40.mdx [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+> > 2 is not the correct prime.
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:38]:
```suggestion
2 is not the correct prime.
```

File: docs/upgrades/locks/ez_40.mdx [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+Possible digits for second parameter: 2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53, 59, 61, 67, 71, 73, 79, 83, 89, 97
+</details>
+
+## Example unlock
+
+<details>
+<summary>Spoilers for c002's solutions</summary>
+```
+
+> > user.fakeloc123 \{ ez_40: "release", c002_complement: "purple" }
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:57]:
```suggestion
>> user.fakeloc123 { ez_40: "release", c002_complement: "purple" }
```

File: docs/upgrades/locks/ez_40.mdx [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+> > WARNING: BINMAT security shell inactive. Intelligent defense system offline.
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:58]:
```suggestion
WARNING: BINMAT security shell inactive. Intelligent defense system offline.
```

File: docs/upgrades/locks/ez_40.mdx [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+> > LOCK_UNLOCKED
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:59]:
```suggestion
LOCK_UNLOCKED
```

File: docs/upgrades/locks/ez_40.mdx [hunk 16]
```diff
@@ -0,0 +1,65 @@
...
+> > System <user> breached.
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:60]:
```suggestion
System <user> breached.
```

File: docs/upgrades/locks/ez_40.mdx [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+> > Connection terminated.
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:61]:
```suggestion
Connection terminated.
```

**Review by seanmakesgames — CHANGES_REQUESTED** [review]

File: docs/upgrades/locks/ez_40.mdx [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+### Stats
+
+This upgrade has no additional stats and only spawns at 'kiddie' rarity.
+
+## Behavior
+
+When encountered, the ez_40 will ask for a string on its first parameter, and a digit (number) on its second parameter.
+
+<details>
+<summary>Spoilers for ez_40's parameters</summary>
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:18]:
```suggestion
```

parameter info is not in the spoiler policy -- should not be in a spoiler block.

File: docs/upgrades/locks/ez_40.mdx [hunk 19]
```diff
@@ -0,0 +1,65 @@
...
+>> user.fakeloc123 { ez_40: "release" }
+LOCK_ERROR
+Required unlock parameter ez_prime is missing.
+
+>> user.fakeloc123 { ez_40: "release", ez_prime: 2 }
+LOCK_ERROR
+2 is not the correct prime.
+
+```
+</details>
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:41]:
```suggestion
```

**Review by seanmakesgames — CHANGES_REQUESTED** [review]

File: docs/upgrades/locks/ez_40.mdx [hunk 20]
```diff
@@ -0,0 +1,65 @@
...
+
+Possible digits for second parameter: 2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53,
+59, 61, 67, 71, 73, 79, 83, 89, 97
+
+</details>
+
+## Example unlock
+
+<details>
+  <summary>Spoilers for c002's solutions</summary>
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:54]:
```suggestion
  <summary>Spoilers for ez_40's solutions</summary>
```

File: docs/upgrades/locks/ez_40.mdx [hunk 21]
```diff
@@ -0,0 +1,65 @@
...
+
+user.fakeloc123 { ez_40: "release", ez_prime: 2 }
+LOCK_ERROR
+2 is not the correct prime.
+```
+
+## Solving
+
+<details>
+  <summary>Spoilers for c002's solutions</summary>
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:42]:
```suggestion
  <summary>Spoilers for ez_40's solutions</summary>
```

**Review by seanmakesgames — CHANGES_REQUESTED** [review]

File: docs/upgrades/locks/ez_40.mdx [hunk 22]
```diff
@@ -0,0 +1,65 @@
...
+
+</details>
+
+## Example unlock
+
+<details>
+  <summary>Spoilers for c002's solutions</summary>
+
+```
+user.fakeloc123 { ez_40: "release", c002_complement: "purple" }
```
Comment (seanmakesgames) [inline docs/upgrades/locks/ez_40.mdx:57]:
```suggestion
>> user.fakeloc123 { ez_40: "release", ez_prime: 43 }
```

**Review by seanmakesgames — CHANGES_REQUESTED** [review]

**Review by kbinreallife — COMMENTED** [review]
✅

**Review by seanmakesgames — APPROVED** [review]
✅

**Pull request merged**